src/django_bootstrap5/templates/django_bootstrap5/widgets/radio_select.html

```diff
@@ -6,7 +6,7 @@
             <div>{{ group }}</div>{% endif %}
         {% for option in options %}
             <div class="form-check">
-                <input class="form-check-input {{ server_side_validation_class }}"
+                <input class="{% filter bootstrap_trim %}form-check-input {{ server_side_validation_class }}{% endfilter %}"
```

Comment:
I don't know if this is a bug or not, but `assertHTMLEqual` now sees leading and trailing spaces as relevant. Therefore, we need to trim.

Reply:
Is there only one place where trailing spaces can occur?

While this solution fixes the test failure, I think it's worth considering alternatives, for example this syntax:

`{% bootstrap_class "form-check-input" server_side_validation_class %}`

Reply:
That's better, done.